llvm/lib/DWARFLinker/DWARFStreamer.cpp

```diff
@@ -859,13 +859,19 @@ void DwarfStreamer::emitLineTablePrologueV5IncludeAndFileTable(
   for (auto Include : P.IncludeDirectories)
     emitLineTableString(P, Include, DebugStrPool, DebugLineStrPool);
 
+  bool InlineSources =
+      std::any_of(P.FileNames.begin(), P.FileNames.end(), [](auto File) {
```

Comment:
Should this be `auto&`  to avoid potentially making a copy?

Reply:
Nit: Here we would generally prefer STLExtras range-based algorithms over std ones:

`any_of(P.FileNames, [](){...})`

Reply:
Oh that's great!